File: dbflow-core/src/main/java/com/raizlabs/android/dbflow/annotation/MultipleManyToMany.java
package com.raizlabs.android.dbflow.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Description: Provides ability to add multiple {@link ManyToMany} annotations at once.
 */
@Retention(RetentionPolicy.SOURCE)
@Target(ElementType.TYPE)
public @interface MultipleManyToMany {

    ManyToMany[] value();
}

File: dbflow-core/src/main/java/com/raizlabs/android/dbflow/annotation/MultipleManyToMany.kt
package com.raizlabs.android.dbflow.annotation

/**
 * Description: Provides ability to add multiple [ManyToMany] annotations at once.
 */
@Retention(AnnotationRetention.SOURCE)
@Target(AnnotationTarget.CLASS, AnnotationTarget.FILE)
annotation class MultipleManyToMany(vararg val value: ManyToMany)
